src/components/organism/AboutUsDetailsSection/AboutUsDetailsSection.tsx

```diff
@@ -1,66 +1,128 @@
+    },
+    {
+      featureHeading: 'Advocating for Social Justice and Equality: ',
+      featureDescription:
+        'We actively work towards creating an inclusive society by advocating for the rights and dignity of all individuals, regardless of gender, caste, religion, or socio-economic background, fostering an environment of equality and justice.',
     },
   ];
 
   const aboutUsParagraph = () => {
     return (
-      <div className="container">
-        <h1 className="text-center m-4 fw-bold">
-          <span className="font-color-orange">About</span> Rise India Foundation
-        </h1>
-        <p>
-          Rise India Foundation is a team of enthusiastic individuals from
-          diverse backgrounds working towards conscientizing the underprivileged
-          citizen and empowering the deprived and marginalized communities
-          leading to improvement in their livelihoods through various programs
-          and activities. Rise India Foundation is a team of enthusiastic
-          individuals from diverse backgrounds working towards conscientizing
-          the underprivileged citizen and empowering the deprived and
-          marginalized communities leading to improvement in their livelihoods
-          through various programs and activities.
-        </p>
-      </div>
+      <section>
+        <div className='container py-5'>
+          <h2 className='text-center fw-bold pb-4 jumbotron-heading'>
+            <span className='font-color-orange'>About</span> Rise India
+            Foundation
+          </h2>
+          <p>
+            Rise India Foundation is a team of enthusiastic individuals from
+            diverse backgrounds working towards conscientizing the
+            underprivileged citizen and empowering the deprived and marginalized
+            communities leading to improvement in their livelihoods through
+            various programs and activities. Rise India Foundation is a team of
+            enthusiastic individuals from diverse backgrounds working towards
+            conscientizing the underprivileged citizen and empowering the
+            deprived and marginalized communities leading to improvement in
+            their livelihoods through various programs and activities.
+          </p>
+        </div>
+      </section>
     );
   };
   const displayInfo = () => {
     return aboutRiseIndiaKeyFeature.map((keyFeature, index) => {
       return (
-        <AboutUsCustomeCardInfo
-          number={index}
+        <AboutUsCustomerCard
+          key={index}
           heading={keyFeature.heading}
           description={keyFeature.description}
+          headingColor={keyFeature.headingColor}
+          backgroundColor={keyFeature.backgroundColor}
         />
       );
     });
   };
+
+  const OurCommitment = () => {
```

Comment:
Keep everything(HTML or UI) under one return statement rather splitting it as smaller function inside a component

Reply:
Removed all functions and added code into one return statement as suggested